climada_petals/hazard/wildfire.py

```diff
@@ -635,7 +662,8 @@ def _firms_centroids_creation(df_firms, res_data, centr_res_factor):
         -------
         centroids : Centroids
         """
-        centroids = Centroids.from_pnt_bounds((df_firms['longitude'].min(), \
+        centroids = Centroids()
+        centroids.set_raster_from_pnt_bounds((df_firms['longitude'].min(), \
```

Comment:
Also here, this is deprecated.
```suggestion
        centroids = Centroids.from_pnt_bounds((df_firms['longitude'].min(), \
```